src/libmongoc/src/mongoc/mongoc-util.c

```diff
@@ -310,6 +310,8 @@ _mongoc_wire_version_to_server_version (int32_t version)
       return "5.3";
    case 17:
       return "6.0";
+   case 21:
```

Comment:
```suggestion
   case WIRE_VERSION_7_0:
```
May be worth considering replacing literals here with their macro equivalents.